app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb

```diff
@@ -0,0 +1,85 @@
+module ManageIQ::Providers
+  class Hawkular::MiddlewareManager::AlertManager
+    require 'hawkular/hawkular_client'
+
+    def initialize(ems)
+      @alerts_client = ems.connect.alerts
```

Comment:
Yes it is equivalent, the connect.alerts is a wrapper of the same alerts_client.
Perhaps it could be good to unify that code in the future.
Thanks for pointing it out.